test/services/LibraryService.test.ts

```diff
@@ -385,6 +385,48 @@ describe('LibraryService', () => {
         });
     });
   });
+
+  describe('$submit', () => {
+    it('returns 201 status with populated content location when provided correct headers and FHIR Library', async () => {
+      await supertest(server.app)
+        .post('/4_0_1/Library/$submit')
+        .send({resourceType: 'Library', status: 'draft'})
+        .set('Accept', 'application/json+fhir')
```

Comment:
Are we ever planning to use this Accept header? If not, I'd remove it from the unit tests